extensions/interactions/EndExploration/directives/end-exploration-validation.service.ts

```diff
@@ -59,22 +59,17 @@ export class EndExplorationValidationService {
           type: AppConstants.WARNING_TYPES.ERROR,
           message: 'Recommended exploration ID must be a string.'
         });
+        break;
       }
-    }
-    // Validate invalid exploration ids given in author recommended
-    // explorations ids.
-    let warningMessageElement = document.querySelector(
-      'oppia-interactive-end-exploration .oppia-error-message-text span'
-    );
-    if (warningMessageElement) {
-      let warningMessage = warningMessageElement.textContent;
-      if (warningMessage) {
+      if (recommendedExplorationIds[i].length === 0) {
```

Comment:
Can you add a test for the false case too?

Reply:
Sure! Seems like the first test `should not have warnings for no answer groups or no default outcome` inherently tests the false case. I've renamed it to `should not have warnings when the end exploration object is valid`. Is that okay?

Reply:
Is it possible to write the test description from an end-user perspective?

Reply:
Sounds good. Thanks!